.github/workflows/MultiXscale:compile_lammps.yml

```diff
@@ -0,0 +1,29 @@
+name: MultiXscale-compile_lammps
+run-name: Compile new Plugin OBMD
+on:
+  push:
+    paths:
+      - '.github/**'
+      - 'src/OBMD/**'
+      - 'examples/OBMD/**'
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - ubuntu-24.04-arm
+          - ubuntu-24.04
+    steps:
+    - uses: actions/checkout@v4
+    - uses: eessi/github-action-eessi@v3
+    - name: set up EasyBuild
+      run: |
+        module load EESSI-extend/2023.06-easybuild
+        eb --show-config
+        echo "Get EasyConfig"
+        eb .github/workflows/LAMMPS_OBMD-foss-2023a-software-commit.eb --software-commit ${{ github.sha }} --include-easyblocks-from-commit 3d295a926318468e3e8fdd99fdcfea02cd796403
+        module avail LAMMPS
+        module load LAMMPS/${{ github.sha }}-foss-2023a-kokkos-dev_OBMD
+        ls $EBROOTLAMMPS
```

Comment:
```suggestion
        echo $EBROOTLAMMPS
        ls $EBROOTLAMMPS
```

Reply:
This is not actually _checking_ that the installation comes from where you think, that's what I suggested the grep